torch/_inductor/inductor_prims.py

```diff
@@ -87,6 +89,7 @@ def eager_force_stride(input_tensor: Tensor, stride) -> Tensor:
         self, indices, values, accumulate
     ),
     doc="Unsafe index_put_ (doesn't issue device asserts)",
+    mutates_args=("self",),
```

Comment:
We can infer this from the schema -- feel free to just parse the schema (torch._C.parse_schema). If you're a pytorch developer writing code that's inside pytorch you probably know about alias annotations